tests/test_storageclass_deletion.py

```diff
@@ -0,0 +1,140 @@
+"""
```

Comment:
@RazTamir Can you review this PR. I removed the other codes and made a single test case out of it